sdk/storage/azure-storage-blob/README.md

```diff
@@ -195,6 +195,54 @@ async for blob in container.list_blobs():
 print(blob_list)
 ```
 
+## Optional Configuration
+
+Optional keyword arguments that can be passed in at the client and per-operation level. 
+
+### Retry Policy configuration
+
```

Comment:
Do we want to give user a hint were these keyword arguments should be passed to? like an example ExponentialRetry or sub class of StorageRetryPolicy? @annatisch what do you think 😃

Reply:
It already says in the next line that these arguments should be passed in at client instantiation :)

Reply:
oooh I didn't check the code and assume ExponentialRetry will not take those parameters automatically.